src/Sylius/Behat/Element/Admin/ExchangeRate/FormElement.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Admin\ExchangeRate;
+
+use Sylius\Behat\Behaviour\SpecifiesItsField;
+use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
+
+final class FormElement extends BaseFormElement implements FormElementInterface
+{
+    use SpecifiesItsField;
+
+    public function isFieldDisabled(string $fieldName): bool
+    {
+        return null !== $this->getElement($fieldName)->getAttribute('disabled');
+    }
+
+    public function getRatio(): string
+    {
+        return $this->getElement('ratio')->getValue();
+    }
+
+    public function hasFormValidationError(string $expectedMessage): bool
```

Comment:
There is a `getValidationErrors` method in the `Crud\FormElement`. Maybe you can use this method.